plasma_framework/test/endToEndTests/FastExit.e2e.test.js

```diff
@@ -192,7 +200,11 @@ contract(
                                 rlpDepositTx,
                                 depositInclusionProof,
                                 depositUtxoPos,
-                                { from: bob, value: this.startStandardExitBondSize },
+                                {
+                                    from: bob,
+                                    value: this.startStandardExitBondSize.add(this.processExitBountySize),
```

Comment:
This addition can be done in `setupContracts`, named `startStandardExitTxValue` or other, better name.